class.jetpack-jitm.php

```diff
@@ -215,7 +207,7 @@ function dismiss( $id, $feature_class ) {
 			$hide_jitm[ $feature_class ] = array( 'last_dismissal' => 0, 'number' => 0 );
 		}
 
-		$number = $hide_jitm[ $feature_class ]['number'];
+		$number = $hide_jitm[ $feature_class ][ 'number' ];
```

Comment:
Just to nitpick a bit, these changes are incorrect:

![image](https://user-images.githubusercontent.com/1883296/39473207-a0e502d4-4d1b-11e8-86b0-ec49d3cb316c.png)